.env.example

```
@@ -51,4 +51,13 @@ VERIFIER_TIMEOUT_SECONDS=2
 # Health check timeout (seconds)
 HEALTH_CHECK_TIMEOUT_SECONDS=2
 
+# Redis Caching Configuration
+CACHE_ENABLED=false                          # Enable Redis-based response caching
+REDIS_URL=localhost:6379                     # Redis server address
+REDIS_PASSWORD=                              # REQUIRED in production! Leave empty for dev
```

dotenv-linter (4.0.0)

[warning] [SpaceCharacter] The line has spaces around equal sign

(SpaceCharacter)